CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- We added functionality to handle arXiv ID in `PdfContentImporter` and implemented related test case. [#12000](https://github.com/JabRef/jabref/issues/12000)
```

Comment:
These comments are user facing. A user does not see `PdfContentImporter` in the UI. We are updating the docs at https://github.com/JabRef/user-documentation/pull/537 - and the updated section should be linked.